components/variations/service/variations_field_trial_creator.h

```diff
@@ -189,7 +189,9 @@ class VariationsFieldTrialCreator {
 
   SeedType seed_type() const { return seed_type_; }
 
+#if !BUILDFLAG(IS_COBALT)
```

Comment:
Instead of changing the whole protected/public scaffolding, how about in
l.210:

```
#if BUILDFLAG(IS_COBALT)
 public:
#endif
  virtual void ApplyFieldTrialTestingConfig(base::FeatureList* feature_list);
#if BUILDFLAG(IS_COBALT)
 protected:
#endif
```

?
It's a larger diff but I think the intent is clearer and if this file
changes in ToT, it'd be easier for a rebaser to move the cobalt-code
around to fit.

Reply:
We actually need every function in the protected block. I was trying to avoid repeating all 4 function definitions twice in the file. Please let me know if you still think this is better

Reply:
How so? I must be missing something because I see e.g. 
`GetPlatform()` in l.197 is not used in this PR.